lmdeploy/pytorch/nn/moe.py

```diff
@@ -35,32 +35,47 @@ def __init__(self,
                  renormalize: bool = False,
                  dtype: Optional[torch.dtype] = None,
                  device: Optional[torch.device] = None,
-                 all_reduce: bool = True):
+                 all_reduce: bool = True,
+                 enable_ep: bool = False):
```

Comment:
How to activate `enable_ep`?

Reply:
Since it has no benefits on mixtral, I decided to disable it until we have DP support. You can manually enable it here.